pkg/odo/cli/pipelines/bootstrap.go

```diff
@@ -75,7 +75,16 @@ func (io *BootstrapParameters) Run() error {
 	if err != nil {
 		return err
 	}
-	log.Success("Bootstrapped GitOps sucessfully.")
+	log.Success("Bootstrapped Openshift resources (GitOps) sucessfully. \n",
+		"Next Steps\n",
+		"1. Bring the bootstrapped environment up. \n",
```

Comment:
I think we should shorten to this to something like...

"Next Steps\n",
"Deploying your new Pipelines configuration https://github.com/rhd-gitops-example/docs/tree/master/journey/day1#bringing-the-bootstrapped-environment-up"

Reply:
The numbered steps don't have links to say what those things mean.